src/index.js

```diff
@@ -1,5 +1,6 @@
 import express from 'express';
 import cors from 'cors';
+import validate from './middlewares/validate';
```

Comment:
Well done, I think you can just import this to **index.js** in the middleware folder and export it from there, this will be a better way of exporting middleware functions because we can be sure that all middleware is being exported from index.js. This helps in a case where we have more than one middleware apart from **validate.js** in the middleware folder, we can just refer to that middleware by importing the index.js in the middleware folder then use it as needed in the codebase. or what do you think @Oliver-ke, let me know if you understand what i am suggesting..

Reply:
I think he did ex2what you just explained src/index.  That is the entry point file now?

Reply:
I was actually talking about middleware/index.js, in a case where we have **validate.js**, **somethingelse.js**, we can import them into **middleware/index.js**, and export the functions in each from there, so that if we need the middleware for other parts of the codebase we just import from middleware/index.js and not have to be calling middleware/validate.js, and middleware/somethingelse.js.. do you get me @funmi5

Reply:
@oyewoas hm, I totally prefer the way he did his, also you have a point but I still really don't get you.

Reply:
> @oyewoas I think he did ex2what you just explained src/index. That is the entry point file now?

I was actually talking about middleware/index.js, in a case where we have validate.js, somethingelse.js, we can import them into middleware/index.js, and export the functions in each from there, so that if we need the middleware for other parts of the codebase we just import from middleware/index.js and not have to be calling middleware/validate.js, and middleware/somethingelse.js.. do you get me @funmi5

Reply:
> @oyewoas hm, I totally prefer the way he did his, also you have a point but I still really don't get you.

its not a pressing thing. and it works all good...

Reply:
The concern raised by @oyewoas has not been addressed yet and it is a valid based on our directory structure of the project. When are you planning to implement that?